src/Avalonia.Themes.Fluent/Controls/SplitButton.xaml

```diff
@@ -0,0 +1,373 @@
+                <Setter Property="UseSystemFocusVisuals" Value="{StaticResource UseSystemFocusVisuals}" />
+                <Setter Property="FocusVisualMargin" Value="-3" />-->
+                <Setter Property="Template">
+                  <ControlTemplate>
+                    <!-- Switched from Grid to Border to optimize and support Foreground/Background properties -->
+                    <Border x:Name="RootBorder"
+                            Background="Transparent">
+                      <ContentPresenter x:Name="ContentPresenter"
+                                        BorderBrush="{TemplateBinding BorderBrush}"
+                                        BorderThickness="{TemplateBinding BorderThickness}"
+                                        Content="{TemplateBinding Content}"
+                                        ContentTemplate="{TemplateBinding ContentTemplate}"
+                                        Padding="{TemplateBinding Padding}"
+                                        HorizontalContentAlignment="{TemplateBinding HorizontalContentAlignment}"
+                                        VerticalContentAlignment="{TemplateBinding VerticalContentAlignment}" />
+                    </Border>
+                  </ControlTemplate>
+                </Setter>
+              </Style>
+              <Style Selector="Button:disabled">
+                <Setter Property="Background" Value="{DynamicResource SplitButtonBackgroundDisabled}" />
+                <Setter Property="Foreground" Value="{DynamicResource SplitButtonForegroundDisabled}" />
+              </Style>
+            </Border.Styles>
+            <Grid>
+              
+              <Grid.ColumnDefinitions>
+                <ColumnDefinition Width="*"
+                                  MinWidth="{DynamicResource SplitButtonPrimaryButtonSize}" />
+                <ColumnDefinition Width="1" />
+                <ColumnDefinition Width="{DynamicResource SplitButtonSecondaryButtonSize}" />
+              </Grid.ColumnDefinitions>
+
+              <!-- Changed from Grid to Rectangle since Avalonia does not support Background on Grid directly -->
+              <Rectangle x:Name="PrimaryBackgroundRectangle"
+                         Fill="{TemplateBinding Background}" />
+
+              <Rectangle x:Name="SecondaryBackgroundRectangle"
+                         Fill="{TemplateBinding Background}"
+                         Grid.Column="2" />
+
+              <!-- Changed from Grid to Border to support Background/Border/CornerRadius properties -->
+              <Border x:Name="Border"
+                      Grid.ColumnSpan="3"
+                      BorderBrush="{TemplateBinding BorderBrush}"
+                      BorderThickness="{TemplateBinding BorderThickness}"
+                      CornerRadius="{TemplateBinding CornerRadius}" />
+
+              <Button x:Name="PART_PrimaryButton"
```

Comment:
Whoops, I renamed this adding the "PART_" prefix that was standard with WPF. However, I didn't update the names in all the styles below... surprised that wasn't noticed in testing but will fix.

Reply:
This was fixed by renaming.